lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -123,19 +125,25 @@ public boolean isConnected() {
 
   @Override public Location getLastLocation(LostApiClient client) {
     throwIfNotConnected(client);
+    Location location = null;
     try {
-      return service.getLastLocation();
+      location = service.getLastLocation();
     } catch (RemoteException e) {
-      throw new RuntimeException(e);
+      Log.e(TAG, "Error occurred trying to get last Location " + e.getStackTrace());
+    } finally {
+      return location;
```

Comment:
As @westnordost pointed out below with availability, this will also return `null` if an error is encountered.

This behavior is consistent will play services.